openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -108,6 +111,32 @@ def student_view(self, context=None):
         """
         fragment = Fragment()
 
+        # Head dependencies
+
+        discussion_vendor_js = ['js/jquery.autocomplete.js', 'js/src/tooltip_manager.js']
```

Comment:
@e-kolpakov Yup, I was planning on doing that -- that's what I meant when I said the changes to `DiscussionXBlock` needed some cleaning up :)

Good news is, `DiscussionCourseXBlock` is already taking care of delivering the assets it depends on, see [xblock_discussion_course.py#L40](https://github.com/edx-solutions/edx-platform/blob/ziafazal/fix-broken-features/openedx/core/lib/xblock_builtin/xblock_discussion_course/xblock_discussion_course.py#L40), so there's a chance we won't have to update it again.